plugins/module_utils/config_loader.py

```diff
@@ -31,7 +31,6 @@ def _load_config_file(self, config_file):
                     "properties": {
                         "token": {"type": "string"}  # token must be a string
                     },
-                    "required": ["token"],  # token field is required
```

Comment:
If this is a temporary change, I would suggest creating an issue on Github to keep track of this change and then remember to restore.

Reply:
I think "temporary" might've been the wrong word - I don't think it should be required given how things work right now, its valid for the config file to not have the token.  

But in general I need to do some more digging into how the authentication flow works from the api user perspective w/ flightctl